### ISSUE TITLE
Install Not Working on any theme or icon pack

### ISSUE THREAD

**Issue by jollygreen68** [vinceliuice/Orchis-theme#467]
Hello, The dark theme is NOT working, nor installing using your terminal commands. I have Ubuntu 24.04. I have Gnome Tweaks installed. I have Gnome Extensions installed. I have dark theme enabled in my Ubuntu settings. I have Dark Compact Theme chosen from Gnome Tweaks and yet it's still white?! 

Plus, you can't install the theme using the command ./install.sh you keep telling everyone to use. That doesn't work at all. In fact, none of your themes or icons install using your instructions. Every time I try a theme or icon install, It just says: No such file or directory. It does this with all of your themes. None of them install using Terminal and the commands you keep saying to use. I have no idea how to get this thing to install properly? I have no idea how to install certain tweaks or color options. And even when using Gnome Tweaks to install the theme, it's not completely dark. It only has certain elements of my DE that are dark, then the rest are light. Pleas help, this so frustrating. Here's a screenshot of my desktop with "White Sur Dark Theme installed. See how the Files are white and not black. Thank you

![Screenshot from 2024-08-31 16-22-14](https://github.com/user-attachments/assets/42b5183c-332a-47a5-b8fc-1bf7b3b8a1eb)

**Comment by vinceliuice** [comment]
Gnome Tweaks can not change gtk 4.0 app's theme anymore since gnome > 40.0

**Comment by vinceliuice** [comment]
You can run `./install.sh -c dark -l` to install the dark one for gtk 4.0 app

**Comment by jollygreen68** [comment]
That does nothing. Anytime I type in ./install.sh or add anything at the end, it just says: NO SUCH FILE OR DIRECTORY. It does not work. 

I've also added the dependencies you mentioned: gnome-themes-extras. It says I already have it. 

I then add gtk2-engines-murrine. It says this: Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
gtk2-engines-murrine is already the newest version (0.98.2-4).
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.

I then add sassc. It says this: Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
sassc is already the newest version (3.6.1+20201027-2).
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.

And here is what it says ANYTIME I type in ./install.sh

![Screenshot from 2024-09-01 11-38-48](https://github.com/user-attachments/assets/fc2bfb01-6279-4fb1-89f2-8d8945088342)

**Comment by vinceliuice** [comment]
After sassc installed run it again

**Comment by vinceliuice** [comment]
Make sure which theme you want install, if you want install `Orchis-dark` then run `./install.sh -c dark -l` at `Orchis-gtk-theme` source folder, if you want install `WhiteSur-Dark` then run `./install.sh -c Dark -l` at `WhiteSur-gtk-theme` source folder

**Comment by jollygreen68** [comment]
Again, not sure what you mean by run ./install.sh -c dark -l at Orchis-gtk-theme source folder. But I'm taking a wild guess and assuming you mean, the GTK4 folder inside the Orhis Dark Theme folder in my Home folder? So I did that, I opened up the Orchis Dark Theme folder and then right clicked on the GTK4 Folder to bring up a terminal and typed in exactly what you said and again, it says No such file or directory. Here's a screen shot of my Home folder opened up to the Orchis Dark Theme folder, selected GTK4 folder, then right clicked to open a terminal, typed in what you said and this is what it spits out:
![Screenshot from 2024-09-01 18-24-17](https://github.com/user-attachments/assets/4c23a51a-de3d-4bb6-9e18-788cba55b7ae)

**Comment by vinceliuice** [comment]
Ddownload from github, you are not using the source files

**Comment by vinceliuice** [comment]
Run this command at the same place with install.sh file is

**Comment by vinceliuice** [comment]
![Screenshot from 2024-09-02 12-14-34](https://github.com/user-attachments/assets/1bac1196-792b-4fa5-888d-35a4b41d0b02)

**Comment by vinceliuice** [comment]
Download this: https://github.com/vinceliuice/Orchis-theme/archive/refs/tags/2024-09-02.tar.gz

**Comment by jollygreen68** [comment]
OMG, FINALLY! Between a guy I've been speaking with on a Linux forum and piecing together what you're saying, I finally got it to work. But it still doesn't work fully or all the time. It's so glitchy or buggy. Like, when I chose the Icon variant for the Activities button in the top left corner, it changes when I type the command into the terminal IN the Orchis theme folder. But then when I restart the computer, it automatically resets to whatever it wants. 

And the colors of the window manager do not work. It's just all dark all the time or all light. NONE of the colors I chose show up. Like this shows the really cool looking black window manager, but with white inside, where the folders go. I can't get this look no matter what option I chose in Gnome Tweaks or how I install through Terminal in the Orchis Theme folder:
![Orchis Theme BlackWhite](https://github.com/user-attachments/assets/fe011e15-c0bd-4386-ad29-7d5d9e186e7d)

I can't get this look if I chose Light theme purple. It's just ALL white.
![OrchisPurple](https://github.com/user-attachments/assets/925dcc39-9874-494d-9c70-2a3e4e758df8)

Another problem. No matter how many times I've installed it through Terminal, by typing the commands in under the Orchis theme folder, I can not get the Macos buttons to install. They never show up anywhere, even though the commands says it installed it. 

Another problem is that my Dash to Dock dock constantly changes from lines with rounded corners to just ROUND sides. It looks so stupid and yet I never change anything and it just does it on it's own. WHY? How do I get it stop doing this. How do I get it to look like it should, with only small rounded corners, instead of two round circle shaped sides? 

Here is a photo of what it looks like often:
![Screenshot from 2024-09-03 19-22-34](https://github.com/user-attachments/assets/2dba3044-33a7-4912-b217-7c0430ffd5fb)

And here is what I want it to look like all the time:
![orchis dock](https://github.com/user-attachments/assets/403ec78c-d268-4d92-80a5-902b2350aeb7)

Thank you for helping so far. It's been extremely difficult to be honest. You leave out a TON of information about how to do this, especially for people that don't know Linux. It's very frustrating, but glad it's sort of working now. Even if it's not completely like how it should be. But thank you. Hopefully you can explain how to fix these other problems I'm having now.

**Comment by vinceliuice** [comment]
Run `./install.sh --tweaks dock` to fix the dock issue

**Comment by vinceliuice** [comment]
Run `./install.sh --tweaks macos` to install macos windows button style

**Comment by jollygreen68** [comment]
Yes, as I stated already above, I ran ./install.sh --tweaks dock to "TRY" to fix the dock issue. Nothing happens. It doesn't get fixed. 

Same thing for the Macos buttons. As I already stated, I ran the command ./install.sh --tweaks macos and terminal "shows" it installing. But then it's not installed. 

I've tried this multiple times. Four times in total to be exact. I run the command, it says it's installing, it doesn't. I log out and restart my computer and nothing has changed. Something is wrong with your theme man. It's just so glitchy and buggy. It's really disappointing, because it's probably one of, if not the most beautiful, clean, aesthetically pleasing themes for a computer desktop I've ever seen. And that's including Macos which is the High Bar in design for computer desktop environments. 

So frustrating and disappointing. I might have to switch to a different distro and see if Orchis works better that way. Or maybe just give up on Orchis, and go with something else that isn't so glitchy, buggy and doesn't work.

**Comment by vinceliuice** [comment]
Run `./install.sh --tweaks macos -l` forget that gtk 4.o app need -l option

**Comment by vinceliuice** [comment]
And try: `sudo ./install.sh --tweaks dock` if your dock extension installed in system dir

**Comment by jollygreen68** [comment]
So I just decided to get off of Ubuntu all together. After reading more and talking with others, what is most likely going on, is that Ubuntu and Gnome are the problem, not your theme. Seems like Ubuntu with Gnome are glitchy and buggy and so I switched over to Linux Mint with Cinnamon DE. It's much better, way more stable, way more customizable out of the box, and yet I can still add your themes, icons and folders with no problems or crashes. So I'm using the White Sur Dark theme you made. It installed right away in Linux Mint and looks fantastic. I do have some questions though, but will ask you over in that Theme, so that my questions and your answers are there for posterity, in case others have similar questions that need answered.

Even though it was very difficult getting detailed help from you, I do want to applaud the effort you made to continually try to help me. I do appreciate that a lot! I just wish you would put all this very important info on HOW to install the theme and how to type in the commands, where to type in the commands, etc in the ReadMe section. It's far too generalized and assumes that people know how to do any of this. Which a lot of people probably do not know. And that makes it extremely frustrating and next to impossible to figure out how to install it all and get it setup how you want. Which is a shame, because your themes and icons are literally the best out there for Linux! 

I saw some of your other beautiful themes and they all look so amazing. You have an eye for great aesthetics and a few of your themes are literally what I would create if I knew how to write code and actually create this stuff. But alas, I'm not a very computer literate person and don't have the time. But thank you for all your hard work and all the amazing themes and icons/etc you have created for free on your own time for everyone! I think I'm going to download a few other themes/icon packs and periodically switch things up, as they all look so amazing in their own way. Best
